docs/source/features/search/basic_search.rst

```diff
@@ -0,0 +1,118 @@
+..
+ Copyright 2021 Graviti. Licensed under MIT License.
+
+##############
+ Basic Search
+##############
+
+TensorBay supports basic search based on different commits.
+
+.. literalinclude:: ../../../../docs/code/basic_search.py
+      :language: python
+      :start-after: """Authorize a Dataset Client Instance"""
+      :end-before: """"""
+
+****************
+ BasicSearchJob
+****************
+
+TensorBay SDK allows create, get or list :class:`~tensorbay.client.job.BasicSearchJob` via :class:`~tensorbay.client.version.BasicSearch`.
+
+Create
+======
+
+A BasicSearchJob can be created by :func:`~tensorbay.client.version.BasicSearch.create_job`
+
+.. literalinclude:: ../../../../docs/code/basic_search.py
+      :language: python
+      :start-after: """Create Job"""
+      :end-before: """"""
+
+.. note::
+    ``conjunction``: The logical conjunction between search filters, which includes "AND" and "OR".
+
+.. note::
+    ``unit``:The unit of basic search. There are two options:
+        * "FILE": Get the data that meets search filters;
+        * "FRAME": If at least one data in a frame meets search filters, all data in the frame will be get. This option only works on fusion dataset.
+
+.. note::
+    ``filters``: The list of basic search criteria whose format is (key, operator, value, label_type).
+        * "key": The factor for filtering, is a member of ["segment", "size", "withLabel", "frame", "sensor", "category", "attribute", "dataRemotePath"].
+        * "operator": The operation for filtering, is a member of ["like", "in", "=", ">", "<", ">=", "<="].
+        * "value": Reference value for filtering.
+        * "label_type"：The label type it belongs to when filtering by category and attribute.
+
+        There are also some limits on operations of specific factor, and it lists as following:
+
+        .. list-table::
+           :widths: auto
+           :header-rows: 1
+
+           * - key
```

Comment:
add the allowed types for the value. such as:
"in" -> List
"size" -> ">"... -> int